reference/orders.v2.oas2.yml

```diff
@@ -3905,15 +3909,19 @@ components:
         external_id:
           description: 'ID of the order in another system. For example, the Amazon Order ID if this is an Amazon order.This field can be updated in a /POST, but using a /PUT to update the order will return a 400 error. The field ''external_id'' cannot be written to. Please remove it from your request before trying again. It can not be overwritten once set.'
           example: null
-          type:
-            - string
-            - 'null'
+          oneOf:
+            - type: string
+            - type: null
         external_source:
-          description: 'For orders submitted or modified via the API, using a PUT or POST operation, you can optionally pass in a value identifying the system used to generate the order. For example: `POS`. Otherwise, the value will be null.'
+          description: |-
+            The value used to identify an external system that generated the order and submitted it to BigCommerce via the Orders API.
+            * When supplying the value, we recommend combining the type of system and vendor, e.g. ERP (Acumatica) or POS (Square).
```

Comment:
* When supplying the value, we recommend combining the type of system and vendor, e.g., ERP (Acumatica) or POS (Square).